target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -275,6 +275,15 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
+	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x00"
```

Comment:
What's this 0x00 for? a led that blinks for no port at all?

Reply:
Sorry, typing mistake. I fixed it.
Leds are not really a big concern, as this is an outdoor device (will one really go to check the leds in the middle of a typhoon ?)